### PR TITLE
Add polymer behavior & polymer behavior implement

### PR DESCRIPTION
**Pull request by VandeurenGlenn**

### CODE REVIEW

File: snippets/polymer-behavior-implement.cson [hunk 1]
```diff
@@ -0,0 +1,14 @@
+".text.html":
+  "polymer behavior implement":
+    "prefix": "pbi"
+    "body": """
+    <script>
```
Comment (robdodson) [inline snippets/polymer-behavior-implement.cson]: Would you mind removing the script tags from both of these snippets?

**Comment by VandeurenGlenn** [comment]
Sure, no problem!

**Comment by robdodson** [comment]
Would you mind doing a rebase to squash the commits together?

**Comment by VandeurenGlenn** [comment]
Done, I also changed html to script highlighting.

**Comment by robdodson** [comment]
Great thank you!

**Comment by VandeurenGlenn** [comment]
Thanks for you're time & you're welcome!

**Pull request merged**